systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicUtils.java

```diff
@@ -384,7 +384,8 @@ public static void waitForReplicaChangeStatusNotPresent(String namespaceName, St
                     }
                     successCounter[0] = 0; // reset counter if condition not met
                     return false;
-                }, TestConstants.GLOBAL_CRUISE_CONTROL_TIMEOUT);
+                    // The timeout change to 10 minutes is needed due to time that CC needs to compute local cluster model before doing changes in Kafka
+                }, TestConstants.GLOBAL_CRUISE_CONTROL_TIMEOUT_LONG);
```

Comment:
Wouldn't be sufficient to tune the configuration of CC as it is done in `TopicReplicasChangeST`? I guess there were similar issues. Applies for other timeouts as well

Reply:
Going to try with CC tuning as a global config for all tests

Reply:
It seems it's working fine with the changes. I will keep them, the timeout is set back to the original one.